src/css/goof-demo.css

```diff
@@ -27,6 +28,35 @@
 	color: var(--fraunces-white);
 }
 
+.goof-demo .a-touch-of-class-sticker {
+	position: absolute;
+	top: 15%;
+	right: 9%;
+	width: 12em;
+	height: 12em;
+	transform: rotate(12deg);
+	pointer-events: none;
+}
+
+@media screen and (max-width: 64rem) {
+	.goof-demo .a-touch-of-class-sticker {
+		right: 5%;
+		width: 10em;
+		height: 10em;
+	}
+}
+
+@media screen and (max-width: 48rem) {
+	.goof-demo .a-touch-of-class-sticker {
+		top: 4%;
+		right: 7%;
+		width: 6em;
+		height: 6em;
+		transform: rotate(12deg);
```

Comment:
We don't need to set this and the following line again, right?

Reply:
Whoops!